odpf/predator/v1beta1/metrics_log.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package odpf.predator.v1;
```

Comment:
@arinda-arif package name should be v1beta1

Reply:
thanks @ravisuhag, updated.